pkg/artifactory/resource/security/resource_artifactory_permission_target_test.go

```diff
@@ -121,7 +121,7 @@ const permissionFull = `
 		actions {
 			users {
 				name        = "anonymous"
-				permissions = ["read", "write"]
+				permissions = ["read", "write", "distribute"]
```

Comment:
We have 2 new permissions added and Only 1 added here

Reply:
I am not sure whether it makes actually sense to add here such permissions for the anonymous user.

Reply:
Let's create a new test (non-anon) user in that block of HCL and reference it in the permission target resource. Then we can include all the permissions and check for them, which currently we don't have any.

Reply:
Did this get done?